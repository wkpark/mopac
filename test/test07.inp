  1scf  
 Aniline  Check that atom labels are being output.
  
  C(Ph-NH2)    
  C(Atom 2)    1.4 1             1
  C(Atom 3)    1.4 1 120 1       2 1
  C(Atom 4)    1.4 1 120 1   0 1 3 2 1
  C(Atom 5)    1.4 1 120 1   0 1 4 3 2
  C(Atom 6)    1.4 1 120 1   0 1 5 4 3
  N(Ph-NH2)    1.4 1 120 1 180 1 1 2 3
  H(on N)      1.0 1 109 1   0 1 7 1 2
  H(on N)      1.0 1 109 1 120 1 7 1 8
  H(Ortho)     1.0 1 120 1 180 1 2 3 4
  H(Meta)      1.0 1 120 1 180 1 3 4 5
  H(Para)      1.0 1 120 1 180 1 4 5 6
  H(Meta')     1.0 1 120 1 180 1 5 6 1
  H(Ortho')    1.0 1 120 1 180 1 6 1 2

