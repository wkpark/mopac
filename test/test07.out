 *******************************************************************************
 ** FRANK J. SEILER RES. LAB., U.S. AIR FORCE ACADEMY, COLO. SPGS., CO. 80840 **
 *******************************************************************************

                                MNDO CALCULATION RESULTS


 *******************************************************************************
 *          MOPAC:  VERSION  6.00               CALC'D. 31-OCT-90               
 *   T=      - A TIME OF  3600.0 SECONDS REQUESTED
 *  DUMP=N   - RESTART FILE WRITTEN EVERY  3600.0 SECONDS
 *  1SCF     - DO 1 SCF AND THEN STOP 
 ***********************************************************************040BY040
  1SCF
 Aniline  Check that atom labels are being output.
 

    ATOM   CHEMICAL    BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL      (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                    NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      C(Ph-NH2) 
      2      C(Atom 2)   1.40000  *                                  1
      3      C(Atom 3)   1.40000  *     120.00000  *                 2    1
      4      C(Atom 4)   1.40000  *     120.00000  *    0.00000  *   3    2    1
      5      C(Atom 5)   1.40000  *     120.00000  *    0.00000  *   4    3    2
      6      C(Atom 6)   1.40000  *     120.00000  *    0.00000  *   5    4    3
      7      N(Ph-NH2)   1.40000  *     120.00000  *  180.00000  *   1    2    3
      8      H(on N)     1.00000  *     109.00000  *    0.00000  *   7    1    2
      9      H(on N)     1.00000  *     109.00000  *  120.00000  *   7    1    8
     10      H(Ortho)    1.00000  *     120.00000  *  180.00000  *   2    3    4
     11      H(Meta)     1.00000  *     120.00000  *  180.00000  *   3    4    5
     12      H(Para)     1.00000  *     120.00000  *  180.00000  *   4    5    6
     13      H(Meta')    1.00000  *     120.00000  *  180.00000  *   5    6    1
     14      H(Ortho')   1.00000  *     120.00000  *  180.00000  *   6    1    2


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        0.0000    0.0000    0.0000
     2         C        1.4000    0.0000    0.0000
     3         C        2.1000    1.2124    0.0000
     4         C        1.4000    2.4249    0.0000
     5         C        0.0000    2.4249    0.0000
     6         C       -0.7000    1.2124    0.0000
     7         N       -0.7000   -1.2124    0.0000
     8         H       -0.0439   -1.9671    0.0000
     9         H       -1.2722   -1.2580    0.8188
    10         H        1.9000   -0.8660    0.0000
    11         H        3.1000    1.2124    0.0000
    12         H        1.9000    3.2909    0.0000
    13         H       -0.5000    3.2909    0.0000
    14         H       -1.7000    1.2124    0.0000
  H: (MNDO):  M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)       
  C: (MNDO):  M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)       
  N: (MNDO):  M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)       


          RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 18


            INTERATOMIC DISTANCES
0
                  C  1       C  2       C  3       C  4       C  5       C  6
 ------------------------------------------------------------------------------
     C    1   0.000000
     C    2   1.400000   0.000000
     C    3   2.424871   1.400000   0.000000
     C    4   2.800000   2.424871   1.400000   0.000000
     C    5   2.424871   2.800000   2.424871   1.400000   0.000000
     C    6   1.400000   2.424871   2.800000   2.424871   1.400000   0.000000
     N    7   1.400000   2.424871   3.704052   4.200000   3.704052   2.424871
     H    8   1.967636   2.440210   3.834868   4.623286   4.392236   3.246559
     H    9   1.967636   3.064925   4.259737   4.623286   3.981532   2.664771
     H   10   2.088061   1.000000   2.088061   3.328663   3.800000   3.328663
     H   11   3.328663   2.088061   1.000000   2.088061   3.328663   3.800000
     H   12   3.800000   3.328663   2.088061   1.000000   2.088061   3.328663
     H   13   3.328663   3.800000   3.328663   2.088061   1.000000   2.088061
     H   14   2.088061   3.328663   3.800000   3.328663   2.088061   1.000000
0
                  N  7       H  8       H  9       H 10       H 11       H 12
 ------------------------------------------------------------------------------
     N    7   0.000000
     H    8   1.000000   0.000000
     H    9   1.000000   1.637686   0.000000
     H   10   2.622975   2.234138   3.299552   0.000000
     H   11   4.507771   4.471476   5.088199   2.400000   0.000000
     H   12   5.200000   5.605882   5.605882   4.156922   2.400000   0.000000
     H   13   4.507771   5.277783   4.686077   4.800000   4.156922   2.400000
     H   14   2.622975   3.585006   2.637536   4.156922   4.800000   4.156922
0
                  H 13       H 14
 ------------------------------
     H   13   0.000000
     H   14   2.400000   0.000000

 -------------------------------------------------------------------------------
  1SCF
 Aniline  Check that atom labels are being output.
 


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                               MNDO    CALCULATION
                                                       VERSION  6.00
                                                       31-OCT-90               




          FINAL HEAT OF FORMATION =         46.59633 KCAL


          TOTAL ENERGY            =      -1072.07051 EV
          ELECTRONIC ENERGY       =      -4406.91122 EV
          CORE-CORE REPULSION     =       3334.84070 EV

          IONIZATION POTENTIAL    =          8.92643
          NO. OF FILLED LEVELS    =         18
          MOLECULAR WEIGHT        =     93.128


          SCF CALCULATIONS  =                1
          COMPUTATION TIME =   5.820 SECONDS





    ATOM   CHEMICAL    BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL      (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                    NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      C(Ph-NH2) 
      2      C(Atom 2)   1.40000  *                                  1
      3      C(Atom 3)   1.40000  *     120.00000  *                 2    1
      4      C(Atom 4)   1.40000  *     120.00000  *    0.00000  *   3    2    1
      5      C(Atom 5)   1.40000  *     120.00000  *    0.00000  *   4    3    2
      6      C(Atom 6)   1.40000  *     120.00000  *    0.00000  *   5    4    3
      7      N(Ph-NH2)   1.40000  *     120.00000  *  180.00000  *   1    2    3
      8      H(on N)     1.00000  *     109.00000  *    0.00000  *   7    1    2
      9      H(on N)     1.00000  *     109.00000  *  120.00000  *   7    1    8
     10      H(Ortho)    1.00000  *     120.00000  *  180.00000  *   2    3    4
     11      H(Meta)     1.00000  *     120.00000  *  180.00000  *   3    4    5
     12      H(Para)     1.00000  *     120.00000  *  180.00000  *   4    5    6
     13      H(Meta')    1.00000  *     120.00000  *  180.00000  *   5    6    1
     14      H(Ortho')   1.00000  *     120.00000  *  180.00000  *   6    1    2


            INTERATOMIC DISTANCES
0
                  C  1       C  2       C  3       C  4       C  5       C  6
 ------------------------------------------------------------------------------
     C    1   0.000000
     C    2   1.400000   0.000000
     C    3   2.424871   1.400000   0.000000
     C    4   2.800000   2.424871   1.400000   0.000000
     C    5   2.424871   2.800000   2.424871   1.400000   0.000000
     C    6   1.400000   2.424871   2.800000   2.424871   1.400000   0.000000
     N    7   1.400000   2.424871   3.704052   4.200000   3.704052   2.424871
     H    8   1.967636   2.440210   3.834868   4.623286   4.392236   3.246559
     H    9   1.967636   3.064925   4.259737   4.623286   3.981532   2.664771
     H   10   2.088061   1.000000   2.088061   3.328663   3.800000   3.328663
     H   11   3.328663   2.088061   1.000000   2.088061   3.328663   3.800000
     H   12   3.800000   3.328663   2.088061   1.000000   2.088061   3.328663
     H   13   3.328663   3.800000   3.328663   2.088061   1.000000   2.088061
     H   14   2.088061   3.328663   3.800000   3.328663   2.088061   1.000000
0
                  N  7       H  8       H  9       H 10       H 11       H 12
 ------------------------------------------------------------------------------
     N    7   0.000000
     H    8   1.000000   0.000000
     H    9   1.000000   1.637686   0.000000
     H   10   2.622975   2.234138   3.299552   0.000000
     H   11   4.507771   4.471476   5.088199   2.400000   0.000000
     H   12   5.200000   5.605882   5.605882   4.156922   2.400000   0.000000
     H   13   4.507771   5.277783   4.686077   4.800000   4.156922   2.400000
     H   14   2.622975   3.585006   2.637536   4.156922   4.800000   4.156922
0
                  H 13       H 14
 ------------------------------
     H   13   0.000000
     H   14   2.400000   0.000000


                  EIGENVALUES

 -43.49328 -36.67916 -33.81430 -30.25909 -24.21722 -22.99438 -18.14000 -17.79776
 -17.70368 -15.46873 -15.06340 -14.81572 -13.37766 -13.00688 -12.70221 -11.39445
  -9.53872  -8.92643   0.27410   0.30820   2.56543   3.66672   4.12383   4.46202
   5.10800   5.21108   5.34118   5.42317   5.78033   5.87188   6.08909   6.20718
   6.37734   6.51441   6.59405


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

         ATOM NO.   TYPE          CHARGE        ATOM  ELECTRON DENSITY
           1          C           0.0232          3.9768
           2          C          -0.0991          4.0991
           3          C          -0.0299          4.0299
           4          C          -0.0779          4.0779
           5          C          -0.0342          4.0342
           6          C          -0.0656          4.0656
           7          N          -0.2301          5.2301
           8          H           0.1128          0.8872
           9          H           0.1178          0.8822
          10          H           0.0550          0.9450
          11          H           0.0549          0.9451
          12          H           0.0556          0.9444
          13          H           0.0559          0.9441
          14          H           0.0617          0.9383
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -0.052    -0.086     0.463     0.474
 HYBRID         0.603    -0.115     0.835     1.036
 SUM            0.551    -0.201     1.298     1.424


          CARTESIAN COORDINATES 

    NO.       ATOM               X         Y         Z

     1         C                  0.0000    0.0000    0.0000
     2         C                  1.4000    0.0000    0.0000
     3         C                  2.1000    1.2124    0.0000
     4         C                  1.4000    2.4249    0.0000
     5         C                  0.0000    2.4249    0.0000
     6         C                 -0.7000    1.2124    0.0000
     7         N                 -0.7000   -1.2124    0.0000
     8         H                 -0.0439   -1.9671    0.0000
     9         H                 -1.2722   -1.2580    0.8188
    10         H                  1.9000   -0.8660    0.0000
    11         H                  3.1000    1.2124    0.0000
    12         H                  1.9000    3.2909    0.0000
    13         H                 -0.5000    3.2909    0.0000
    14         H                 -1.7000    1.2124    0.0000


          ATOMIC ORBITAL ELECTRON POPULATIONS

   1.18778   0.91028   0.89432   0.98442   1.19126   0.90985   0.94153   1.05642
   1.19264   0.93934   0.91664   0.98128   1.19322   0.92509   0.93568   1.02386
   1.19333   0.92462   0.93379   0.98248   1.18851   0.94148   0.90143   1.03417
   1.58405   1.21645   1.02137   1.40825   0.88724   0.88224   0.94501   0.94512
   0.94439   0.94413   0.93832
  CHANNEL 12 OPENED



 TOTAL CPU TIME:             6.28 SECONDS

 == MOPAC DONE ==
